docker/docker-compose-cassandra.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.3"
```

Comment:
I would call this file `docker-compose-delta.yml` instead of cassandra. 
It is about running delta with all its third party dependencies.

Reply:
Yep, I will see the naming again to include delta but soon we will have a docker compose file with postgres which will be a little different.

Reply:
I keep the cassandra part because at some point jdbc is coming